### PR TITLE
chore(pr): PR template proposal for partner extension

### PR DESCRIPTION
**Pull request by gregorycat**
Add a PR tempalte for the Extension PR.

This template will be used when partner want to deploy a new version of it's extension or when he want update extension details (name/description/ whitelisted customers/...)

### CODE REVIEW

File: template-react-ts/.github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<!--- Choose your type of bump of version you want to deploy -->
+## Type of bump
+Choose one above (if applicable)
+- [ ] Major
+- [ ] Minor
+- [ ] Patch
+
+<!--- If your extension is private add customer names and ids above -->
+## Whitelisted custmers
```
Comment (aureldent) [inline template-react-ts/.github/pull_request_template.md:12]: typo:  ## Whitelisted custmers  =>  ## Whitelisted customers

File: template-react/.github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+<!--- Choose your type of bump of version you want to deploy -->
+## Type of bump
+Choose one above (if applicable)
+- [ ] Major
+- [ ] Minor
+- [ ] Patch
+
+<!--- If your extension is private add customer names and ids above -->
+## Whitelisted custmers
```
Comment (aureldent) [inline template-react/.github/pull_request_template.md:12]: typo: ## Whitelisted custmers => ## Whitelisted customers

**Review by aureldent — COMMENTED** [review]
some typos

File: template-react-ts/.github/pull_request_template.md (hunk 1 above)
Comment (maximevast) [reply template-react-ts/.github/pull_request_template.md:12]:
```suggestion
## Whitelisted customers
```

File: template-react/.github/pull_request_template.md (hunk 2 above)
Comment (maximevast) [reply template-react/.github/pull_request_template.md:12]:
```suggestion
## Whitelisted customers
```

**Review by maximevast — APPROVED** [review]

**Pull request merged**